lib/connection.cpp

```diff
@@ -1070,7 +1093,12 @@ QByteArray Connection::accessToken() const { return d->data->accessToken(); }
 
 QtOlm::Account* Connection::olmAccount() const
```

Comment:
This should not even be in the API if E2EE is off, maybe? (because how do we know wth `QtOlm::Account` is)

Reply:
Agree. It's interesting that it looks like I'm not using proposed API anywhere yet %) Should be double-checked why do I proposed it later.